src/azure-cli/azure/cli/command_modules/vm/tests/latest/example_steps.py

```diff
@@ -0,0 +1,69 @@
+# --------------------------------------------------------------------------
```

Comment:
Is this file used?

Reply:
No, it is generated. Do we need to delete it?